### PR TITLE
[CPU] Native 1D models support

### PR DESCRIPTION
**Pull request by alexey-varyzgin**
### Details:
 - *1D Convolution support was added*
 - *1D Group Convolution support was added*
 - *Zero points for 1D Convolution was added*
 - *1D Pooling support was added*
 - *1D Reorders was added*

https://github.com/openvinotoolkit/oneDNN/pull/84

### Tickets:
 - *62683* 
 - *44483*
 - *38612*

### CODE REVIEW

**Comment by dmitry-gorokhov** [comment]
@antonvor Please start the review

File: inference-engine/src/low_precision_transformations/src/weightable_layer_transformation.cpp [hunk 1]
```diff
@@ -96,7 +96,7 @@ bool WeightableLayerTransformation::canBeTransformed(const TransformationContext
             }
 
             const auto rankVal = rank.get_length();
-            if ((rankVal != 4) && (rankVal != 5)) {
+            if ((rankVal < 3) && (rankVal > 5)) {
```
Comment (antonvor) [inline inference-engine/src/low_precision_transformations/src/weightable_layer_transformation.cpp:99]: This condition is always false
```suggestion
            if ((rankVal < 3) || (rankVal > 5)) {
```

File: inference-engine/tests/functional/plugin/cpu/single_layer_tests/convolution.cpp [hunk 2]
```diff
@@ -184,6 +184,15 @@ const SizeVector numOutChannels_Gemm = {6 };
 /* ============= Convolution params (blocked and nspc layout) ============= */
 const SizeVector numOutChannels = { 64, 63 };
 
+/* ============= Convolution params (1D) ============= */
+const std::vector<SizeVector> kernels1d = { {3} };
```
Comment (antonvor) [inline inference-engine/tests/functional/plugin/cpu/single_layer_tests/convolution.cpp:188]: what about `kernel == {1}`? I think we should add this case too

File: inference-engine/tests/functional/plugin/cpu/single_layer_tests/group_convolution.cpp [hunk 3]
```diff
@@ -184,6 +184,13 @@ const SizeVector numGroups_Blocked = {2, 4};
 const SizeVector numOutChannels_DW = {32};
 const SizeVector numGroups_DW = {32};
 
+/* ============= GroupConvolution params (1D) ============= */
+const std::vector<SizeVector> kernels1d = { {3} };
```
Comment (antonvor) [inline inference-engine/tests/functional/plugin/cpu/single_layer_tests/group_convolution.cpp:188]: add case `kernel == 1` please

**Review by antonvor — CHANGES_REQUESTED** [review]

File: inference-engine/src/low_precision_transformations/src/weightable_layer_transformation.cpp (hunk 1 above)
Comment (alexey-varyzgin) [reply inference-engine/src/low_precision_transformations/src/weightable_layer_transformation.cpp]: Done

File: inference-engine/tests/functional/plugin/cpu/single_layer_tests/group_convolution.cpp (hunk 3 above)
Comment (alexey-varyzgin) [reply inference-engine/tests/functional/plugin/cpu/single_layer_tests/group_convolution.cpp]: Done

**Review by antonvor — APPROVED** [review]

**Comment by antonvor** [comment]
@dmitry-gorokhov could you please finally review?

**Comment by dmitry-gorokhov** [comment]
@antonvor please fix remaining comment and resolve conflicts

File: inference-engine/src/mkldnn_plugin/mkldnn_graph_optimizer.cpp [hunk 4]
```diff
@@ -572,7 +572,7 @@ void MKLDNNGraphOptimizer::FuseConvolutionAndZeroPoints(MKLDNNGraph &graph) {
         ptrdiff_t OC = weightsConstantDims[0 + groupOffset];
         ptrdiff_t IC = weightsConstantDims[1 + groupOffset];
         ptrdiff_t KD = weightsConstantDims.size() == (5 + groupOffset) ? weightsConstantDims[weightsConstantDims.size() - 3] : 1;
-        ptrdiff_t KH = weightsConstantDims[weightsConstantDims.size() - 2];
+        ptrdiff_t KH = node->getInputShapeAtPort(0).getRank() > 3 ? weightsConstantDims[weightsConstantDims.size() - 2] : 1;
```
[the remark this answers is not in the document]
Comment (antonvor) [reply inference-engine/src/mkldnn_plugin/mkldnn_graph_optimizer.cpp]: fixed

**Pull request merged**